layout/AppMenuitem.js

```diff
@@ -75,19 +76,20 @@ const AppMenuitem = (props) => {
             ) : null}
 
             {item.to && !item.items && item.visible !== false ? (
-                <Link legacyBehavior href={item.to} replace={item.replaceUrl} target={item.target}>
-                    <a onClick={(e) => itemClick(e)} className={classNames(item.class, 'p-ripple', { 'active-route': isActiveRoute })} target={item.target} tabIndex="0">
+                <Link legacyBehavior pathname={item.to} href={item.to} replace={item.replaceUrl} target={item.target}>
+                    <a id={item.label} pathname={item.to} href={item.to} onClick={(e) => itemClick(e)} className={classNames(item.class, 'p-ripple', { 'active-route': isActiveRoute })} target={item.target} tabIndex="0">
```

Comment:
Do we need both the <Link> and <a> to have the href property? I think only one of them should have it, and I don't think the <a> is nececcsarry because the <Link> probably creates an <a> which means in the final dom you'll probably see <a><a></a></a> which probably breaks accessibily or something.